src/peft/tuners/adaption_prompt/layer.py

```diff
@@ -17,9 +17,129 @@
 import torch
 import torch.nn as nn
 import torch.nn.functional as F
+from typing import Optional, Tuple, Union
```

Comment:
No need for `Tuple`, we can use `tuple[...]`.

Reply:
Ah, looks like `make style` also changed up some other files (making the import order alphabetic) hope that is not too messy.